server-common/src/main/java/org/apache/gravitino/server/authorization/GravitinoAuthorizerProvider.java

```diff
@@ -40,7 +41,21 @@ private GravitinoAuthorizerProvider() {}
    * @param serverConfig Gravitino server config
    */
   public void initialize(ServerConfig serverConfig) {
-    // TODO
+    if (gravitinoAuthorizer == null) {
+      synchronized (this) {
+        if (gravitinoAuthorizer == null) {
+          boolean enableAuthorization = serverConfig.get(Configs.ENABLE_AUTHORIZATION);
+          if (enableAuthorization) {
+            // TODO
+          } else {
+            gravitinoAuthorizer = new IgnoreAuthorizer();
+          }
+          if (gravitinoAuthorizer != null) {
```

Comment:
In which scenario will the `gravitinoAuthorizer` be null?

Reply:
When Configs.ENABLE_AUTHORIZATION is set to true, it will be null because JcasbinAuthorizer has not been implemented yet. If we don't check for null before the implementation of JcasbinAuthorizer, a null pointer exception will occur here.